test/integration/asserts.py

```diff
@@ -0,0 +1,98 @@
+"""A library with assertation helpers for unit tests."""
```

Comment:
I wonder - do we have a good reason to maintain our own library as compared to using something like:

https://docs.python.org/2/library/unittest.html

Note - it would be ok to address this in a follow-up.

Reply:
This came to be out of unawareness of that library existing. I agree it would be good to move to that.
I would prefer to punt this to an issue though, to not increase size of this PR (and keep it a no-op)